src/libImaging/Quant.c

```diff
@@ -789,6 +789,47 @@ resort_distance_tables(
     return 1;
 }
 
+/* build_distance_tables() below used to call qsort() from stdlib.h.
+ * qsort() is not guaranteed stable, so different implementations can
+ * give different results, causing images created on Linux and Windows
+ * to differ. Shell sort is also not stable, but having the implementation
+ * included here will give consistent results across platforms. -rdg- */
+
+/*  ssort()  --  Fast, small, qsort()-compatible Shell sort */
+/*  By Raymond Gardner. Released here under PIL license.    */
```

Comment:
```suggestion
/* ssort()  --  Fast, small, qsort()-compatible Shell sort */
/* By Raymond Gardner. Released here under PIL license.    */
```
This makes it line up with the previous block.